### ISSUE TITLE
gradle plugins version didn't updated

### ISSUE THREAD

**Issue by flshbt** [littlerobots/version-catalog-update-plugin#120]
**Describe the issue**
version update of normal project deplencies running fine, but using (toml declarted) plugins and version, do not update to the latest stable version. Is there a special trick or is it a bug?
And this trick will be increase the plugin versions in the libs.toml? I must be allways increase the plugin versions manually.

`
[versions]
plugin-gradle-versions = "0.47.0"
plugin-kotlin-version = "1.9.10" <-- this will not be upgrade to lates version
plugin-nebula-publish = "20.1.0" <-- this will not be upgrade to lates version
plugin-nebula-release = "16.2.2"  <-- this will not be upgrade to lates version
plugin-version-catalog = "0.8.1"
my-lib = "2.0.33"   <-- will be upgrade to the lates version

[libraries]
... some libs

[plugins]
plugin-gradle-versions = { id = "com.github.ben-manes:gradle-versions-plugin", version.ref = "plugin-gradle-versions" }
plugin-kotlinus-jvm = { id = "org.jetbrains.kotlin:kotlin-gradle-plugin", version.ref = "plugin-kotlin-version" }
plugin-nebula-publish = { id = "com.netflix.nebula:nebula-publishing-plugin", version.ref = "plugin-nebula-publish" }
plugin-nebula-release = { id = "com.netflix.nebula:nebula-release-plugin", version.ref = "plugin-nebula-release" }
plugin-version-catalog = { id = "nl.littlerobots.vcu:plugin", version.ref = "plugin-version-catalog" }
`

**Comment by hvisser** [comment]
This is basically the same as #119.

TL;DR dependencies are determined by the dependency versions plugin. If that doesn't see an update, the TOML file isn't updated. It won't see updates if the dependency isn't actually used in your project either. If you can reproduce the dependency versions plugin not finding issues, I'd advice to file an issue against that plugin.
